tests/plugins/pastefromwordimage/helpers.js

```diff
@@ -0,0 +1,112 @@
+/* bender-tags: editor,pastefromwordimage */
+/* bender-ckeditor-plugins: pastefromwordimage,ajax */
+
+( function() {
+	'use strict';
+
+	bender.test( {
+		cacheBuster: function() {
+			return ( new Date() ).getTime().toString() + Math.floor( Math.random() * 1000000 ).toString();
+		},
+
+		'test img tags extraction from html string': function() {
+			var sourceLength = 5,
+				i;
+
+			for ( i = 0; i < sourceLength; i++ ) {
+				bender.tools.testInputOut( 'source' + i, function( input, output ) {
+					var expectedResult = JSON.parse( output.replace( /<!--/, '' ).replace( /-->/, '' ) ),
```

Comment:
I like the creative use of `testInputOutput` to use it for array comparison using JSON. However why are you wrapping expected results with a HTML comments?

Keep it simple.

Reply:
I was pretty sure that incorrect HTML markup might break something in bender, but it seems to work fine. Most probably I tested it not on `textarea`s but on regular `div`s. :/